packages/server/logging/expressLogging.ts

```diff
@@ -1,7 +1,75 @@
 import { logger } from './logging'
+import { randomUUID } from 'crypto'
 import HttpLogger from 'pino-http'
+import { IncomingMessage } from 'http'
+import { NextFunction, Response } from 'express'
+import pino, { SerializedResponse } from 'pino'
 
 export const LoggingExpressMiddleware = HttpLogger({
   logger,
-  autoLogging: false
+  autoLogging: true,
+  genReqId: (req) => {
```

Comment:
if you extracted the function out from the `DetermineRequestIdMiddleware`, to take the existing id first and fall back to the header, than the guid, you could pass that function in here to the `genReqId` argument and reuse it in the middleware